### PR TITLE
Add support for the ignoreRepeat parameter

### PR DESCRIPTION
**Pull request by alex-mechler**
cc @iwiznia 

Needed for https://github.com/Expensify/Expensify/issues/158814

Adds support for the new ignoreRepeat parameter. Also makes BedworkWorkerManager use it. 

Tests in https://github.com/Expensify/Web-Expensify/pull/32496

### CODE REVIEW

**Review by rafecolton — APPROVED** [review]
LGTM, adding @iwiznia as a reviewer as well.

**Review by iwiznia — DISMISSED** [review]

**Comment by iwiznia** [comment]
I approved, but you can't merge this, since you need to update the version

File: composer.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
     "name": "expensify/Bedrock-PHP",
     "description": "Bedrock PHP Library",
     "type": "library",
-    "version": "1.9.15",
+    "version": "1.9.17",
```
Comment (alex-mechler) [inline composer.json:5]: Skipped 1.9.16 because it was tagged but never updated here

**Comment by alex-mechler** [comment]
Tagged & updated in composer.json

**Review by iwiznia — APPROVED** [review]

**Review by rafecolton — APPROVED** [review]

**Pull request merged**